creator-node/src/services/sync/secondarySyncFromPrimary.ts

```diff
@@ -296,7 +315,7 @@ const handleSyncFromPrimary = async ({
      */
 
     const maxClockRecordId = Math.max(
-      ...fetchedClockRecords.map((record) => record.clock)
+      ...fetchedClockRecords.map((record: any) => record.clock)
```

Comment:
I use a lot of `any`s for our DB models because we don't have an typescript visibility around there. Once we type our DB models then we can change these to proper DB types

Reply:
👍